File: MPI/mpicheck/tests/functions/3.1/MPI_Aint_diff.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Aint var_0;
    MPI_Aint var_1;
    MPI_Aint ret;
    /* calls */
    ret = MPI_Aint_diff(var_0, var_1);
    ret = PMPI_Aint_diff(var_0, var_1);
    return 0;
}

File: MPI/mpicheck/tests/functions/3.1/MPI_Aint_diff.f90

        
        program main
        use mpi
        
        INTEGER(KIND=MPI_ADDRESS_KIND) var_0
       INTEGER(KIND=MPI_ADDRESS_KIND) var_1
       INTEGER(KIND=MPI_ADDRESS_KIND) ret
        ret = mpi_aint_diff(var_0, var_1)
       ret = pmpi_aint_diff(var_0, var_1)
        end program main
    